libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -444,10 +444,20 @@ dictionary BitcoinAddressData {
     string? message;
 };
 
+dictionary LnUrlSuccessData {
```

Comment:
This only applies to LNURL-pay, so a more accurate name is `LnUrlPaySuccessData` IMO.